scripts/script.js

```diff
@@ -0,0 +1,162 @@
+let tasks = [];
+
+// Get Time and date in the required format  27/07/2022 @ 9:38 pm
+let getTime = () => {
```

Comment:
This is a really good approach to create smaller helper functions to format the date and time. However, I think that the name is a bit deceiving. It is always a good idea to name the function to explicitly predict what's happening in the function. Since we are getting and formatting both date and time in this, it will be better if we change this name to `getDateAndTime`